README.zh.md

```diff
@@ -60,10 +68,10 @@ OpenYurt 支持最高版本为1.18的 Kubernetes 。使用更高版本的 Kubern
 ```bash
 git clone https://github.com/openyurtio/openyurt.git
 cd openyurt
-make WHAT=cmd/yurtctl
+make build WHAT=cmd/yurtctl
 ```
 
-`yurtctl` 的二进制文件位于_output /bin 目录。如果需要将已存在的 Kubernetes 集群转换为 OpenYurt 集群，你可以使用如下简单的命令(目前支持minikube, kubeadm, ack, kind 四种工具安装的kubernetes集群)：
+`yurtctl` 的二进制文件位于_output /bin 目录。如果需要将已存在的 Kubernetes 集群转换为 OpenYurt 集群，你可以使用如下简单的命令(目前支持minikube, kubeadm, kind 四种工具安装的kubernetes集群)：
```

Comment:
Maybe the number of tools here forgot to change, it should be three after reduce the ack.

Reply:
fixed